src/libs/ValidationUtils.js

```diff
@@ -203,7 +203,9 @@ function meetsAgeRequirements(date) {
     const eighteenYearsAgo = moment().subtract(18, 'years');
     const oneHundredFiftyYearsAgo = moment().subtract(150, 'years');
     const testDate = moment(date);
-    return testDate.isValid() && testDate.isBetween(oneHundredFiftyYearsAgo, eighteenYearsAgo);
+
+    // Only compare the dates (ignore the time), and make the comparison inclusive of the start and end dates
+    return testDate.isValid() && testDate.isBetween(oneHundredFiftyYearsAgo, eighteenYearsAgo, 'day', '[]');
```

Comment:
You mean it doesn't work in that the message is technically wrong? Yeah fair point, I'm trying to thing what's the best way to update it:

- Either we can update just the message to be something like "Must be over 18 years old and less than 150 years old", or
- Split the function `meetsAgeRequirements` into two functions, e.g. `isBelowAgeRequirement` and `isOverAgeRequirement` but that sounds overkill, or
- Change the function `meetsAgeRequirements` to return an int, for example `-1` if the age is < 18, `0` if it's fine, `1` if it's over 150 years

Any preference or alternative ideas?

Reply:
> correct because this is the main issue we're trying to fix and makes user confused.

Oooh, admittedly I read the issue too fast and thought the problem that QA reported was that we were showing the error if you selected the oldest possible date in the date selector. But yes you're right, and we should use use a different error message then.

Splitting `meetsAgeRequirements` into two functions works for me. I'm checking about the new error message now because it sounds weird to say you have to be less than 150 years old.